.github/workflows/types_check.yml

```diff
@@ -0,0 +1,500 @@
+name: Test types support
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  # Allow only one workflow per any non-trunk branch.
+  group: ${{ github.workflow }}-${{ github.ref_name }}-${{ github.ref_name == 'trunk' && github.sha || 'any-sha' }}
+  cancel-in-progress: true
```

Comment:
Given this is a workflow_dispatch only action, do we need this concurrency check?

Reply:
Forgot to remove after workflow testing, to be updated. Thank you!